### PR TITLE
[CHORE] considering X-Forwarded-For on HTTP Logging

### PR DESCRIPTION
**Pull request by nicolastakashi**
* [x] I added CHANGELOG entry for this change.
* [ ] Change is not relevant to the end user.

## Changes

- Considering X-Forwarded-For on HTTP Logging, solves issue #7302 

## Verification

N/A

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
 - [#7233](https://github.com/thanos-io/thanos/pull/7233): UI: Showing Block Size Stats
 - [#7280](https://github.com/thanos-io/thanos/pull/7281): Adding User-Agent to request logs
 - [#7219](https://github.com/thanos-io/thanos/pull/7219): Receive: add `--remote-write.client-tls-secure` and `--remote-write.client-tls-skip-verify` flags to stop relying on grpc server config to determine grpc client secure/skipVerify.
+- [##7302](https://github.com/thanos-io/thanos/pull/7303) Considering the `X-Forwarded-For` header for the remote address in the logs.
```
Comment (MichaHoffmann) [inline CHANGELOG.md:39]: Minor typo

**Review by MichaHoffmann — DISMISSED** [review]
Lgtm

**Review by MichaHoffmann — DISMISSED** [review]

**Comment by nicolastakashi** [comment]
@MichaHoffmann I think it's a flaky test.

**Review by saswatamcode — APPROVED** [review]
LGTM!

**Pull request merged**